resources/conf/runners/Library.py

```diff
@@ -0,0 +1,125 @@
+from browserstack.local import Local
+import json
+import csv
+import os
+from robot.libraries.BuiltIn import BuiltIn
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.keys import Keys
+
+
+bstack_local = 0
+se2lib = BuiltIn().get_library_instance('SeleniumLibrary')
+
+def combine_dict(dict1, dict2):
+    dict_1 = json.loads(str(dict1))
+    dict_2 = json.loads(str(dict2))
+
+    dict_1.update(dict_2)
+
+    return dict_1
+
+
+def get_caps_single(caps_path, username, access):
+
+    with open(caps_path) as f:
+        data = json.load(f)
+
+    single_caps_common = data['tests']['single']['common_caps']
+    single_caps_env = data['tests']['single']['env_caps'][0]
+
+    single_caps_common.update(single_caps_env)
+    single_caps_common['browserstack.user'] = username
+    single_caps_common['browserstack.key'] = access
+
+    return single_caps_common
+
+def get_caps_local(caps_path, username,access):
+
+    with open(caps_path) as f:
+        data = json.load(f)
+
+    caps_common = data['tests']['local']['common_caps']
+    caps_env = data['tests']['local']['env_caps'][0]
+
+    caps_common.update(caps_env)
+    caps_common['browserstack.user'] = username
+    caps_common['browserstack.key'] = access
+
+
+    return caps_common
+
+def start_local(key):
+
+    global bstack_local
+    bstack_local = Local()
+
+
+    bstack_local_args = { "key": key }
+
+    bstack_local.start(**bstack_local_args)
+
+    return bstack_local
+
+def stop_local():
+
+    global bstack_local
+
+    bstack_local.stop()
+
+
+def get_row_item_from_file(filepath, key):
+
+    dirname = os.path.dirname(__file__)
+    filename = os.path.normpath(os.path.join(dirname, filepath))
+
+    with open(filename,'r') as data: 
+        for line in csv.reader(data): 
+                    if line[0] == key:
+                        return line[1]
+
+
+def get_password(username):
+
+    password = get_row_item_from_file('../../data/user.csv', username)
+    return password
+
+def get_product_prices():
+    dirname = os.path.dirname(__file__)
+    filename = os.path.normpath(os.path.join(dirname, '../../data/product.csv'))
```

Comment:
define this string "../../data/product.csv" at the top of the file as a constant